daily-challenges/2024-10/21/khanh5923_21.spec.ts

```
@@ -0,0 +1,36 @@
+        await page.click(`//button[@id='logout-btn']`);
+    });
+    
+    test('khanh5923_21', async ({ page }) => {
+        const locatorTicket = `//div[@id='ticket-section']//h3[text()='${dataTest.ticketName}']/following-sibling`;
+        await page.click(`${locatorTicket}::button`);
+        await page.fill(`//input[@id='ticket-quantity']`, `1`);
+        page.on('dialog', async dialog => {
+            await dialog.accept();
+        });
+        await page.click(`//button[@class='btn-confirm']`);
+        let miniCart = page.locator(`//div[@id="cart-icon"]//span[@id="cart-count"]`);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'miniCart' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)